package/MDAnalysis/analysis/encore/clustering/affinityprop.c

```diff
@@ -0,0 +1,9163 @@
+/* Generated by Cython 0.29.21 */
```

Comment:
This file is not supposed to be checked into git.

Reply:
I removed the file.